1001-2000/LIT1501GadlaS.xml

```diff
@@ -33,17 +44,49 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+        <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Translation"/>
+                </keywords>
+            </textClass>
+         <langUsage>
+           <language ident="en">English</language>
+           <language ident="gez">Gǝʿǝz</language>
+         </langUsage>
+         <abstract>
+<p>The Ethiopic vita of Shenoute of Atripe is, according to its editor, a translation from Arabic. It is divided into 52 chapters,
+each devoted to a particular miracle.</p>
+         </abstract>
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2016-03-21"> Created file from google
             spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: text record</change>
+         <change who="MKr" when="2024-01-26">Added relations, bibliography</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <ab/>
       </body>
    </text>
-</TEI>
\ No newline at end of file
+   <text>
+       <body>
+           <div type="bibliography">
+             <listBibl type="secondary">
+               <bibl><ptr target="bm:Buzi2010EAEShenute"/></bibl>
+             </listBibl>
+             <listBibl type="editions">
+                 <bibl><ptr target="bm:Colin1982Shenoudi"/></bibl>
+             </listBibl>
+               <listRelation>
+                   <relation name="ecrm:P129_is_about" active="LIT1501GadlaS" passive="PRS8823Shenute"/>
+                   <relation name="saws:isAttributedToAuthor" active="LIT1501GadlaS" passive="PRS2743Besa"/>
+                   <relation name="saws:isVersionOf" active="LIT1501GadlaS" passive="LIT2492VitaSi"/>
```

Comment:
I guess that here https://github.com/BetaMasaheft/Documentation/issues/2452 @thea-m suggested that we only leave one record, not two. If the Ethiopic text is just the translation then we usually have one record for all languages. In this case LIT2492VitaSi and the relation should be deleted.

Reply:
`<relation name="betmas:formerlyAlsoListedAs" active="LIT1501GadlaS" passive="LIT2492VitaSi"/>
`